deployment-docs/clouds.md

```diff
@@ -0,0 +1,12 @@
+# Clouds
+
+Taskcluster is a multi-cloud system, and as such expects credentials for several cloud providers.
+The services' Kubernetes deployment can run anywhere; at Mozilla it is deployed in GKE, Google Cloud's Kubernetes offering.
+Task artifacts are stored in Amazon S3, so AWS credentials will be required.
+Services' data is stored in Azure Table Storage, so Azure credentials will be required.
+
+Taskcluster can dynamically provision workers in a variety of clouds.
+You will need appropriate credentials for any clouds you intend to use for workers.
```

Comment:
Later it'd be great to go into more detail on what credentials are appropriate, as this is quite non-obvious, but that's likely outside the scope of these changes

Reply:
Yeah, we need to loop back to that at some point!